populus/assets/defaults.v4.config.json

```diff
@@ -150,7 +150,7 @@
       "$ref": "compilation.backends.SolcCombinedJSON"
     },
     "contracts_source_dir": "./contracts",
-    "import_remappings": []
+    "import_remappings": ["fixtures_remap=/actual/path/to/populus/tests/fixtures_remap"]
```

Comment:
This needs to be removed.  We can't modify old config files.  Always has to be a new config file.